### ISSUE TITLE
mosquitto restart issue during install

### ISSUE THREAD

**Issue by Roysteroonie** [owntracks/recorder#465]
I'm using IONOS VPS and have tried this a couple of times now. It fails at the same stage. Even reimagining my server and starting fresh it appears. Anyone else encountered this? 

![image](https://github.com/owntracks/recorder/assets/24797196/659518d6-9dc6-4ca0-9b4b-8f0672a2d968)

Any help or advice greatly appreciated.

**Comment by jpmens** [comment]
I'm assuming this is using our Quickstart?

Please show the last portion of the mosquitto log file from `/var/log/mosquitto/mosquitto.log`, whether you see errors there.

**Comment by Roysteroonie** [comment]
Thanks for the response @jpmens here is the output. 

1716678864: Error: Unable to load CA certificates. Check cafile "/usr/local/owntracks/tls/cert.crt".
1716678864: OpenSSL Error[0]: error:80000002:system library::No such file or directory
1716678864: OpenSSL Error[1]: error:10000080:BIO routines::no such file
1716678864: OpenSSL Error[2]: error:05880002:x509 certificate routines::system lib
1716678864: mosquitto version 2.0.11 starting
1716678864: Config loaded from /etc/mosquitto/mosquitto.conf.
1716678864: Opening ipv4 listen socket on port 1883.
1716678864: Opening ipv4 listen socket on port 8883.
1716678864: Opening ipv6 listen socket on port 8883.
1716678864: Error: Unable to load CA certificates. Check cafile "/usr/local/owntracks/tls/cert.crt".
1716678864: OpenSSL Error[0]: error:80000002:system library::No such file or directory
1716678864: OpenSSL Error[1]: error:10000080:BIO routines::no such file
1716678864: OpenSSL Error[2]: error:05880002:x509 certificate routines::system lib
1716678864: mosquitto version 2.0.11 starting
1716678864: Config loaded from /etc/mosquitto/mosquitto.conf.
1716678864: Opening ipv4 listen socket on port 1883.
1716678864: Opening ipv4 listen socket on port 8883.
1716678864: Opening ipv6 listen socket on port 8883.
1716678864: Error: Unable to load CA certificates. Check cafile "/usr/local/owntracks/tls/cert.crt".
1716678864: OpenSSL Error[0]: error:80000002:system library::No such file or directory
1716678864: OpenSSL Error[1]: error:10000080:BIO routines::no such file
1716678864: OpenSSL Error[2]: error:05880002:x509 certificate routines::system lib

**Comment by jpmens** [comment]
You have not responded to my query regarding whether this was done with Quicksetup.

The error is clearly marked:

>1716678864: Error: Unable to load CA certificates. Check cafile "/usr/local/owntracks/tls/cert.crt".

it would appear the bootstrap process didn't run correctly, or that you didn't specify an email address in the configuration.yaml file before bootstrapping. Is that it?

**Comment by Roysteroonie** [comment]
Yes via Quickstart, I have my own SSL on the VPS, so maybe wrongly assumed that it would use this? 
Apologies for delayed response as the mails are being treated as spam.

**Comment by jpmens** [comment]
Quickstart expects to take main control of the system, so no. (Didn't we already talk about this regarding HTTP port 80?)

**Comment by Roysteroonie** [comment]
So maybe I'd have to purchase a new domain instead of using a current sub from my main ecommerce website, as that requires the ssl?

**Comment by jpmens** [comment]
There should be no need to purchase a new domain. You ought to be able to create a new record in DNS for your current domain (e.g. owntracks.example.com if your current is example.com) and point its A / AAAA record to the IPv4 / IPv6 address of the machine you intend running OwnTracks quickstart on.
